packages/fxa-settings/src/pages/Pair/Outcome/index.tsx

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import CardHeader from '../../../components/CardHeader';
+import { logPageViewEvent } from '../../../lib/metrics';
+import { HeartsBrokenImage, HeartsVerifiedImage } from '../../../images';
+export enum Outcomes {
+  failure = 'failure',
+  success = 'success',
+}
+
+type PairOutcomeProps = {
+  outcome: Outcomes;
+};
+
+const PairOutcome = ({ outcome }: PairOutcomeProps & RouteComponentProps) => {
```

Comment:
Hmm should this be a component instead, used in a `PairFailure` page and a `PairSuccess` page that we use when we set up the route? I think we could still do something like `<PairOutcome path="/pair/failure outcome={whatevs} />` and link/navigate to it correctly, I think it just feels a little funky since we don't have a page called "Pair/Outcome". I like the idea of this being the component hit at the end of the flow regardless though. Maybe if we can update the stories in Storybook to show the page paths instead of "Outcomes" that would help? What're your thoughts?

Reply:
I had this exact same thought process...and then completely forgot about it 😂 so thank you for thinking through it! You know, I wonder if it's even worth it to make a shared component, given that it's just a CardHeader, a <p>, and an image. What would you think of just completely splitting these out into two components?

Reply:
Maybe the pages should use the `Ready` component... 😜😜 Otherwise known as the `HeaderImageParagraph` component.

Reply:
@millsoper Splitting into two separate pages to match our conventions works for me. :party-parrot:

Reply:
@vpomerleau strong laugh/cry reaction to the above 😂